src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/induction/CiagInductionResponseMapper.kt

```diff
@@ -42,12 +42,8 @@ class CiagInductionResponseMapper(
         inPrisonInterests = inPrisonInterestsMapper.toPrisonWorkAndEducationResponse(inPrisonInterests),
         createdBy = createdBy!!,
         createdDateTime = instantMapper.toOffsetDateTime(createdAt)!!,
-        // The main Induction domain object is immutable (the data that can be changed belongs in associated "child"
-        // objects). However, the CIAG API stores details regarding a prisoner's work aspirations at the root level,
-        // which means that any changes to these values needs to be reflected in the "modified" fields of the root
-        // Induction in the response.
```

Comment:
I was never entirely comfortable with this, as it felt like a fudge to the mapping between our domain model and and the externally facing CIAG API model. I think simply returning the modifiedBy/modifiedDateTime of the main Induction (which is now always updated) makes much more send.